gulpfile.js

```diff
@@ -411,3 +431,32 @@ function repeat(str, num) {
     }
     return resp;
 }
+
+
+// Controllers inheritance configs
+var ctrlConfig = {
+    ctrlRegex: /.*\.controller\.js$/,
+    ctrlNameRegex: /\.controller\(['"]([^'"]+)['"]/,
+    baseDir: config.src + config.base,
+    themeDir: config.src + config.theme,
+};
+
+// Prefix with '_' parent controllers in base theme
+// so that we can use the same name for child controllers in theme
+function inheritControllers(file) {
```

Comment:
lets try to be more descriptive of what this function is doing.  Let's use a name that minimizes the amount of comments we need to add above the function name.  Even with what is here, I need to read thru the code to figure what we are trying to do.